Create keplie_ci.yml

JegorKuz opened the pull request:
Added basic CI pipeline for build.

JegorKuz committed:
Create keplie_ci.yml

Added basic CI pipeline for build.

miked4king commented on .github/workflows/keplie_ci.yml:
```diff
@@ -0,0 +1,45 @@
+
+jobs:
+  build:
+
```
лишний отступ, убери еще в других местах ниже

miked4king commented on .github/workflows/keplie_ci.yml:
```diff
@@ -0,0 +1,45 @@
+        distribution: 'temurin'
+
+    - name: Setup Gradle
+      uses: gradle/actions/setup-gradle@af1da67850ed9a4cedd57bfd976089dd991e2582 # v4.0.0
```
# v4.0.0 - можем убрать, ниже тоже

miked4king commented on .github/workflows/keplie_ci.yml:
```diff
@@ -0,0 +1,45 @@
+      uses: gradle/actions/setup-gradle@af1da67850ed9a4cedd57bfd976089dd991e2582 # v4.0.0
+
+    - name: Build with Gradle Wrapper
+      run: ./gradlew build
```
после билда стоит добавить запуск тестов, мало ли в них кто нибудь накосячит))

JegorKuz replied:
А разве тесты не запускаются уже по дефолту в "./gradlew build"?
Когда мы добавляем "./gradlew test" - это ведь по сути разве что добавляет отдельный лог по тестам, однако при этом ещё и запускает тесты даже в случае когда билд провалился(что со слов AI не рекомендуемая практика)? 
Может тогда стоит изолировать тесты? То есть вместо "./gradlew build" -> "./gradlew assemble" и отдельно тесты, потому что иначе получается что мы прогоняем тесты дважды?

JegorKuz committed:
Added tests into CI.

JegorKuz committed:
Update keplie_ci.yml

исправил ошибку повторного запуска билда, на запуск тестов.

JegorKuz committed:
Update keplie_ci.yml

Убрал пробелы про которые забыл.
Хочу попробовать изолировать тесты в отдельный кружок

JegorKuz committed:
Merge branch 'kelpie-master-stream10' into kelpie-stream10-BJS2-77447

JegorKuz committed:
[UPDATE] Решение для таски BJS2-77447.
Исправил + закоментил не проходящий тест.
